ios/FluentUI.Demo/FluentUI.Demo/Demos/ListItemDemoController_SwiftUI.swift

```diff
@@ -146,72 +148,72 @@ struct ListItemDemoView: View {
                 .resizable()
         }
 
-        @ViewBuilder
-        var listItem: some View {
-            ListItem(title: title,
-                     subtitle: showSubtitle ? subtitle : "",
-                     footer: showFooter ? footer : "",
-                     leadingContent: {
-                         if showLeadingContent {
-                             leadingContent
-                         }
-                     },
-                     trailingContent: {
-                        if showTrailingContent {
-                             switch trailingContentFocusableElementCount {
-                             case 0:
-                                 Text("Spreadsheet")
-                             case 1:
-                                 Toggle("", isOn: $trailingContentToggleEnabled)
-                             default:
-                                 HStack {
-                                     Button {
-                                         showingSecondaryAlert = true
-                                     } label: {
-                                         Text("Button 1")
-                                     }
-                                     Button {
-                                         showingSecondaryAlert = true
-                                     } label: {
-                                         Text("Button 2")
-                                     }
-                                 }
-                             }
-                         }
-                     },
-                     action: !isTappable ? nil : {
-                         showingPrimaryAlert = true
-                     }
-            )
-            .backgroundStyleType(backgroundStyle)
-            .accessoryType(accessoryType)
-            .leadingContentSize(leadingContentSize)
-            .titleLineLimit(titleLineLimit)
-            .subtitleLineLimit(subtitleLineLimit)
-            .footerLineLimit(footerLineLimit)
-            .combineTrailingContentAccessibilityElement(trailingContentFocusableElementCount < 2)
-            .onAccessoryTapped {
-                showingSecondaryAlert = true
-            }
-            .disabled(isDisabled)
-            .alert("List Item tapped", isPresented: $showingPrimaryAlert) {
-                Button("OK", role: .cancel) { }
-            }
-            .alert("Detail button tapped", isPresented: $showingSecondaryAlert) {
-                Button("OK", role: .cancel) { }
-            }
+        func listItem() -> some View {
```

Comment:
Should still be possible to tag this as a `@ViewBuilder` if you remove the `return` later in the function, I think?